package.json

```diff
@@ -73,5 +73,8 @@
     "swarm-icons": "0.1.0",
     "swarm-sasstools": "1.0.9",
     "webpack": "^2.2"
+  },
+  "dependencies": {
+    "swarm-animation": "git+ssh://git@github.com/meetup/swarm-animation.git"
```

Comment:
I think we've decided to treat build-time dependencies as `devDependencies` (i.e. they don't need to be part of the production environment), so this should move up alongside the other `devDependencies`